charts/ndb-operator/templates/deployment-ndb-operator-controller-manager.yaml

```diff
@@ -26,7 +26,7 @@ spec:
           - --health-probe-bind-address=:8081
           - --metrics-bind-address=127.0.0.1:8080
           - --leader-elect
-        image: {{ .Values.image.repository }}:{{ .Values.image.tag | default $.Chart.AppVersion }}
+        image: {{ .Values.image.repository }}:{{ $.Chart.AppVersion }}
```

Comment:
usually we keep tag exemple of customer who are using his own registry and who add tag suffix once image are validated or compressed